mllib/src/test/scala/org/apache/spark/ml/classification/RandomForestClassifierSuite.scala

```diff
@@ -158,7 +159,7 @@ class RandomForestClassifierSuite
   }
 
   test("Fitting without numClasses in metadata") {
-    val df: DataFrame = spark.createDataFrame(TreeTests.featureImportanceData(sc))
+    val df: DataFrame = TreeTests.featureImportanceData(sc).toDF()
```

Comment:
I also agree with this but actually it seems both are fine assuming from this discussion, https://github.com/apache/spark/pull/12452